tests/common/devices/sonic.py

```diff
@@ -1762,6 +1762,42 @@ def get_vlan_intfs(self):
 
         return vlan_intfs
 
+    def get_vlan_brief(self):
+        """
+        Get vlan brief
+        Sample output:
+            {
+                "Vlan1000": {
+                    "interface_ipv4": [],
+                    "interface_ipv6": [],
+                    "members": ['Ethernet0', 'Ethernet1']
+                },
+                "Vlan2000": {
+                    "interface_ipv4": [],
+                    "interface_ipv6": [],
+                    "members": ['Ethernet3', 'Ethernet4']
+                }
+            }
+        """
+        config = self.get_running_config_facts()
+        vlan_brief = {}
+        for vlan_name, members in config["VLAN_MEMBER"].items():
+            vlan_brief[vlan_name] = {
+                "interface_ipv4": [],
+                "interface_ipv6": [],
+                "members": list(members.keys())
+            }
+        for vlan_name, vlan_info in config["VLAN_INTERFACE"].items():
+            if vlan_name not in vlan_brief:
+                continue
+            for prefix in vlan_info.keys():
+                ip = prefix.split('/')[0]
```

Comment:
For this common function, I think we should reserve the prefix. You can remove it in testcase if necessary.

Reply:
Ok, updated